deploy/dlt/Dockerfile

```diff
@@ -17,16 +17,28 @@ ADD compiled_requirements.txt .
 
 # install alpine deps
 RUN apk update &&\
-    apk add --no-cache python3 ca-certificates curl postgresql &&\
-    apk add --no-cache --virtual build-deps build-base automake autoconf libtool python3-dev postgresql-dev libffi-dev linux-headers gcc musl-dev &&\
-    ln -s /usr/bin/python3 /usr/bin/python &&\
+    apk add --no-cache ca-certificates curl postgresql git &&\
+    apk add --no-cache --virtual build-deps build-base automake autoconf libtool python3-dev postgresql-dev libffi-dev linux-headers gcc musl-dev cmake &&\
     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py &&\
     rm get-pip.py &&\
-    pip3 install --upgrade setuptools wheel &&\
-    rm -r /usr/lib/python*/ensurepip &&\
-    pip3 install -r compiled_requirements.txt &&\
+    pip install --upgrade setuptools wheel pip &&\
+    rm -r /usr/lib/python*/ensurepip
+
+# install arrow 17.0.0, usually we would need apache-arrow-dev=17.0.0 but it is not available in alpine 3.20
+# adapt this version to the arrow version you need
+RUN git clone --no-checkout https://github.com/apache/arrow.git /arrow \
```

Comment:
as noted in the comment, alpine always seems to come with one version of each package, i remember something like this from our last company, so I need to build arrow 17 here if I want to use it..